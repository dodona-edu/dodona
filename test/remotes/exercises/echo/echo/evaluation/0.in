proscribable
crystallisation
overfruitful
sterilizable
unforgivingness
snatchy
dropworts
chaufers
hyperbolized
bicentennially
curring
ageratum
preallotment
deputize
anthropopsychic
flustroid
ketones
cantharidian
adenolymphoma
headlongly
linoleums
pseudoracemic
flagpole
extraessential
unclustering
preglobulin
spiderflower
uncorker
antithrombic
lienomedullary
unshrivelled
benison
copiable
infiltering
incivil
turncoat
autosome
outfight
numbnesses
reforestization
mastochondrosis
ridottos
reapproval
grasping
dioicously
synergia
macrauchene
schizospore
nonzero
seminomadism