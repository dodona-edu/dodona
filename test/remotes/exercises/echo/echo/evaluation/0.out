proscribable
crystallisation
overfruitful
sterilizable
unforgivingness
snatchy
dropworts
chaufers
hyperbolized
bicentennially
curring
ageratum
preallotment
deputize
anthropopsychic
flustroid
ketones
cantharidian
adenolymphoma
headlongly
linoleums
pseudoracemic
flagpole
extraessential
unclustering
preglobulin
spiderflower
uncorker
antithrombic
lienomedullary
unshrivelled
benison
copiable
infiltering
incivil
turncoat
autosome
outfight
numbnesses
reforestization
mastochondrosis
ridottos
reapproval
grasping
dioicously
synergia
macrauchene
schizospore
nonzero
seminomadism
----------------------------------------
python input without prompt: true
block count: multi
input block size: 1
output block size: 1 
